src/main/kotlin/com/ampnet/reportservice/enums/SupportedLanguages.kt

```diff
@@ -0,0 +1,5 @@
+package com.ampnet.reportservice.enums
+
+enum class SupportedLanguages {
+    EN, EL, ES
+}
```

Comment:
We must not set supported languages in code, all supported languages should be in json file.